blib2to3/pgen2/tokenize.py

```diff
@@ -107,59 +109,27 @@ def maybe(*choices): return group(*choices) + '?'
 pseudoprog = re.compile(PseudoToken, re.UNICODE)
 single3prog = re.compile(Single3)
 double3prog = re.compile(Double3)
+
+_strprefixes = set(combinations('r', 'R', 'f', 'F') +
+                   combinations('r', 'R', 'b', 'B') +
+                   ['u', 'U'])
```

Comment:
You're missing `'uR', 'UR', 'Ur', 'ur'` which is valid on Python 2.

Reply:
uR right :rofl: I was wondering why tokenize used to accept this in the regex above.

Reply:
Funnily enough, since we are still supporting `u""` in Python 3, not supporting `ur""` is likely just an omission of whoever brought `u""` back :-)